source/expressions.tex

```diff
@@ -3644,10 +3644,9 @@
 
 \pnum
 An lvalue
-of type ``\cvqual{cv1} \tcode{T1}'' can be cast to type ``rvalue
-reference to \cvqual{cv2} \tcode{T2}'' if ``\cvqual{cv2} \tcode{T2}'' is
-reference-compatible with ``\cvqual{cv1}
-\tcode{T1}''\iref{dcl.init.ref}. If the value is not a bit-field,
+of type \tcode{T1} can be cast to type ``rvalue
```

Comment:
We could say "An lvalue of (possibly cv-qualified) type T1 ... " to address Walter's concerns. But it's harder to fit in the "possibly cv-qualified" for T2. I do agree with the intent here that we only write "cv T" if we want to talk about "cv" separately.